pkg/apis/configuration/v1alpha1/types.go

```diff
@@ -30,6 +30,24 @@ type ConfigurationSpec struct {
 	TracingSpec TracingSpec `json:"tracing,omitempty"`
 	// +optional
 	MTLSSpec MTLSSpec `json:"mtls,omitempty"`
+	// +optional
+	Secrets SecretsSpec `json:"secrets,omitempty"`
+}
+
+// SecretsSpec is the spec for secrets configuration
+type SecretsSpec struct {
+	Scopes []SecretsScope `json:"scopes"`
```

Comment:
May be useful to document the input expectation here. For example, are StoreName allowed to be duplicated in the list?

Also, if there are expectation about the input, please add some validation logic to ensure.

Reply:
Will document it and add it as part of docs. If the storeName is repeated, the last configuration will override previously defined configurations.